### PR TITLE
Add payment type and wire up transaction cell

### PR DESCRIPTION
**Pull request by jewsroch**
Added payment type model and pointer from transaction model. (also added in parse)
Wired up TransactionCell to view data and tableView.

### CODE REVIEW

File: PerDiem/PerDiem/Controllers/TransactionsViewController.m [hunk 1]
```diff
@@ -26,6 +29,10 @@ - (void)viewDidLoad {
 
     self.edgesForExtendedLayout = UIRectEdgeNone;
     [self setupNavigationBar];
+
+    // @Todo Fetch Transactions
```
Comment (jewsroch) [inline PerDiem/PerDiem/Controllers/TransactionsViewController.m]: Need to finish this, but wasn't sure how we were planning to organize the models for this common fetch.

File: PerDiem/PerDiem/TransactionCell.m [hunk 2]
```diff
@@ -14,6 +25,27 @@ - (void)awakeFromNib {
...
 
+- (void)setTransaction:(Transaction *)transaction {
+    _transaction = transaction;
+
+    self.dateLabel.text = [self formatDate:self.transaction.transactionDate];
+    self.budgetLabel.text = self.transaction.budget.name;
+    self.paymentTypeLabel.text = self.transaction.paymentType.name;
+    self.summaryLabel.text = self.transaction.summary;
+
+    // @TODO - make common currency formatter method to be used throughout app.
```
Comment (jewsroch) [inline PerDiem/PerDiem/TransactionCell.m]: We should make some common helpers for currency and date formatting. 
Also, we should figure out the best way to easily make all currency strings localized.

Comment (flochtililoch) [inline PerDiem/PerDiem/TransactionCell.m]: here's an example on how to create a currency formatter:

in private interface:

``` objective-c
@property (strong, nonatomic) NSNumberFormatter *amountFormatter;
```

in implementation:

``` objective-c
- (NSNumberFormatter *)amountFormatter {
    if (!_amountFormatter) {
        _amountFormatter = [[NSNumberFormatter alloc] init];
        [_amountFormatter setNumberStyle: NSNumberFormatterCurrencyStyle];
    }
    return _amountFormatter;
}
```

**Comment by flochtililoch** [comment]
:shipit:

File: PerDiem/PerDiem/Controllers/TransactionsViewController.m (hunk 1 above)
Comment (frojasg) [inline PerDiem/PerDiem/Controllers/TransactionsViewController.m]: after merging #13 you can use 

``` objective-c
        [Transaction transaction:^(NSArray *transactions, NSError *error) {
            for (Transaction *transaction in transactions) {
                NSLog(@"%@", budget);
            }
        }];
```

**Pull request merged**